### ISSUE TITLE
Check on RStudio Approval

### ISSUE THREAD

**Issue by achafetz** [USAID-ICPI/WeekendSpecial#50]
Rachel said SA CIO won't install on people's machines since it's no longer approved. Needs to train SI team on Friday to be able to work in R and access our scripts

**Comment by achafetz** [comment]
Check approval list on AIDNET as starter - USAID Software List - https://pages.usaid.gov/M/CIO/approved-software

**Comment by achafetz** [comment]
Actually I was able to access the site @jaliasd (didn't use to be possible)

Here is what the approval list looks like.
![image](https://user-images.githubusercontent.com/8933069/41664114-f75f2580-7472-11e8-89e8-7b533e34f3e8.png)

Version 1.0.136 shown as approved. The messaging say "Approved cfor [sic] general use" so it is approved. The restrictions are around security settings when install. Key message approved.

Looked at the most recent version 1.1.447 to see why it was cancelled.  The note states
> request cancelled - R studio 1.x is already approved 

So, bottom line, any verison 1.x is approved for USAID use.
